Directory.Packages.props

```diff
@@ -16,23 +16,24 @@
     <PackageVersion Include="Microsoft.AspNetCore.Http.Abstractions" Version="[2.1.1,6.0)" />
     <PackageVersion Include="Microsoft.AspNetCore.Http.Features" Version="[2.1.1,6.0)" />
     <PackageVersion Include="Microsoft.CSharp" Version="[4.7.0]" />
-    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="[3.1.0,)" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="[3.1.0,)" />
-    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="[3.1.0,)" />
-    <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="[2.1.0,)" />
 
     <!--
-        Typically, the latest stable version of Microsoft.Extensions.Logging should be used here because:
-        1) Each major version bump will have some new API capabilities (e.g. .NET 6 introduced compile-time logging
+        Typically, for the Microsoft.Extensions.* packages relating to DI Abstractions, Hosting Abstractions, and Logging,
+        the latest stable version should be used because:
+        1) Each major version bump will have some new API capabilities (e.g.For Logging, .NET 6 introduced compile-time logging
           source generation, .NET 8 introduced automatic event id generation).
         2) Each minor version bump is normally security hotfixes or critical bug fixes.
         3) Since version 3.1.0, the .NET runtime team is holding a high bar for backward compatibility on
-          Microsoft.Extensions.Logging even during major version bumps, so compatibility is not a concern here.
+          these packages even during major version bumps, so compatibility is not a concern here.
     -->
-    <PackageVersion Include="Microsoft.Extensions.Logging" Version="8.0.0-rc.2.23479.6" />
 
+    <PackageVersion Include="Microsoft.Extensions.Configuration" Version="8.0.0-rc.2.23479.6" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="8.0.0-rc.2.23479.6" />
+    <PackageVersion Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0-rc.2.23479.6" />
+    <PackageVersion Include="Microsoft.Extensions.Logging" Version="8.0.0-rc.2.23479.6" />
     <PackageVersion Include="Microsoft.Extensions.Logging.Configuration" Version="[3.1.0,)" />
```

Comment:
This package will be updated in a different PR. It's not as straightforward as simply updating this package breaks the build.

Reply:
This was covered in #5020